runtime/minor_gc.c

```diff
@@ -711,10 +717,9 @@ static void minor_gc_leave_barrier
     }
   }
 
-  /* Spin a bit longer, which is far less fruitful if we're waiting on more than
-     one thread */
-  unsigned spins = participating_count == 2
-    ? Max_spins_medium : Max_spins_short;
+  /* Spin a bit longer, which is far less fruitful if we're waiting on
+     more than one thread */
+  unsigned spins = participating_count == 2 ? Max_spins_medium : Max_spins_short;
   SPIN_WAIT_NTIMES(spins) {
```

Comment:
This change to the comment formatting and instruction indentation change is unnecessary; in fact, it makes the instruction 81 characters wide.